website/content/en/docs/ansible/reference/advanced_options.md

```diff
@@ -209,3 +209,56 @@ spec:
     served: true
     storage: true
 ```
+
+## Passing Arbitrary Arguments to Ansible
+
+Users can specify an arbitrary command-line argument to the ```operator-sdk exec-entrypoint``` and ```run --local``` subcommands for Ansible-based Operators via the ```ansible-args``` flag.
+
+**Example**
+
+The example below has ```--tags``` as the arbitrary command-line argument. Tags allow a playbook to run a specific part of the configuration without running the whole playbook. The aribtrary argument is specified as: 
+```
+operator-sdk run local --ansible-args='--tags "configuration,packages"'
+```
+```
+operator-sdk run local --ansible-args='--skip-tags "notification"'
+```
```

Comment:
the run local also will be removed so we need to describe it with the bin instead of run local.